treatment-service/integration-test/docker-compose/kafka/docker-compose.yaml

```diff
@@ -1,5 +1,3 @@
-version: '3'
-
```

Comment:
In Docker Compose v2, this version field is no longer used: https://stackoverflow.com/a/76157215